data/nlu/nlu.md

```diff
@@ -3765,6 +3765,14 @@
 - where to start?
 - yes i wanna know more about rasa [nlu](product)
 - I am trying to build a bot using rasa
+- more about [Rasa open source](product)
+- tell me about [rasa open source](product)
+- i want to know about [rasa Open source](product)
+- get started with [Rasa Open Source](product)
+- [Rasa open source](product)
+- [rasa open source](product)
+- [rasa Open source](product)
+- [Rasa Open Source](product)
```

Comment:
did you check this doesn't cause conflict with `ask_faq_rasaoss` or whatever the faq intent about RAsa open source is

Reply:
In evaluation it does not, it can still differentiate between "is rasa open source" and "rasa open source".

Reply:
interesting 🤔